kotlin/kooby/build.gradle

```diff
@@ -23,16 +22,21 @@ plugins {
 group "benchmark.kooby"
 version "1.0.0"
 mainClassName = "benchmark.AppKt"
-sourceCompatibility = 1.8
+sourceCompatibility = 17
+
+kotlin {
+    jvmToolchain {
+        languageVersion.set(JavaLanguageVersion.of("17"))
```

Comment:
maybe we should have this on all kotlin based framework if this is something overridden by default (the jvm is 17 of dockerfiles)

Reply:
honestly? not sure why we need it when `sourceCompatibility` is there, without it gradle was failing.

Reply:
can you remove it if it is not necessary

Reply:
I'm not a kotlin/gradle user, but without these lines gradle fails.

Reply:
feel free to change to maven of anything you recommend for `kooby` users

Reply:
It's Ok.  Kotlin users prefer gradle.